src/Components/BDCard/BDCard.js

```diff
@@ -70,34 +112,34 @@ function BDCard() {
 		          options={options}
 		        />
 						<section className="select-data">
-							<h6 className="info">03% || $13,470</h6>
-							<h6 className="selection">Legend Title</h6>
+							<h6 className="info">{displayOptionData(true)}</h6>
+							<h6 className="selection">{brokerOption}</h6>
 						</section>
 					</section>
         </section>
       </article>
       <article className="card-segment">
-        <h4>Premium Range</h4>
+        <h4>{cardData.chartTitle}</h4>
 				<section className="chart">
 					<section className="legend">
 						<ul>
-							{createList(['Under $1,000', '$1,000 - $10,000', '$10,000 - $50,000', 'Over $50,000'])}
+							{displayList(false)}
 						</ul>
 					</section>
 					<section className="chart-area">
 		        <Doughnut
-		          data={brokerChart}
+		          data={carrierChart}
 		          options={options}
 		        />
 						<section className="select-data">
```

Comment:
I'm not sure how exactly to get the percentage to show up in the center of the chart. These numbers also need to be formatted correctly